tokio/src/sync/broadcast.rs

```diff
@@ -603,6 +603,88 @@ impl<T> Sender<T> {
         new_receiver(shared)
     }
 
+    /// Returns the number of queued values.
```

Comment:
The documentation should explain that this is the number of messages that at least one channel have not yet received. (And also note that lagged messages are not counted, since the receiver len includes those.)